Sources/GRPCCodeGen/Internal/Translator/ClientCodeTranslator.swift

```diff
@@ -64,6 +64,11 @@
 /// }
 ///```
 struct ClientCodeTranslator: SpecializedTranslator {
+  var accessLevel: SourceGenerator.Configuration.AccessLevel
+
+  init(accessLevel: SourceGenerator.Configuration.AccessLevel) {
+    self.accessLevel = accessLevel
+  }
   func translate(from codeGenerationRequest: CodeGenerationRequest) throws -> [CodeBlock] {
```

Comment:
Leave a space between functions
```suggestion
  }

  func translate(from codeGenerationRequest: CodeGenerationRequest) throws -> [CodeBlock] {
```